src/test/IKImageTransformation.test.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
```

Comment:
if the test filename and component name matches then `IKImageTransformation` makes no sense. We can either use filenames to show what test cases are about or stick with component name convention and club test cases together.

Reply:
These tests have been moved into `IKImage.test.js`